mingw-w64-git/PKGBUILD

```diff
@@ -97,17 +99,23 @@ i686)
 x86_64)
   SHAREDIR=/mingw64/share/git
   ;;
+aarch64)
+  SHAREDIR=/clangarm64/share/git
+  ;;
 esac
 
 prepare () {
 	cd "$srcdir/git" &&
 
 	cat >config.mak <<-EOF
-	USE_ASCIIDOCTOR = YesPlease
+	USE_ASCIIDOCTOR = $([[ "$MSYSTEM" == "CLANGARM64" ]] || echo "YesPlease")
 	EOF
 
-	test -n "$WITHOUT_PDBS" ||
```

Comment:
It would make the code more readable to first move the definition of the compat flags into the `WITHOUT_PDBS` conditional that already exists at the top of the file, in a separate commit, something like this:

```sh
if test -z "$WITHOUT_PDBS"
then
    [...]

    COMPAT_FLAGS="-gdwarf-4 -gstrict-dwarf" # cv2pdb cannot really handle DWARF5 yet
fi
```

Then, a subsequent commit can introduce a clang-specific block _before_ that and have the clang-specific overrides. I have something like this in mind:

```
if test CLANGARM64 = "$MSYSTEM"
then
   [...]
   if test -z "$WITHOUT_PDBS"
   then
       [...]
    fi
elif test -z "$WITHOUT_PDBS"
then
   [...]
fi
```

Side note (read: no actionable item as far as _this_ PR is concerned, but more a reminder to myself): The code comment states that `cv2pdb` does not really handle DWARF5 yet. But I am actually quite certain that https://github.com/rainers/cv2pdb/pull/67 and https://github.com/rainers/cv2pdb/pull/69 addressed that, and that the comment is therefore stale. A subsequent commit should therefore drop those `-gdwarf-4 -gstrict-dwarf` options.